packages/amplify-codegen/commands/codegen/model-introspection.js

```diff
@@ -0,0 +1,24 @@
+const codeGen = require('../../src');
+const { exitOnNextTick } = require('amplify-cli-core');
+const featureName = 'model-introspection';
+const path = require('path');
+
+module.exports = {
+  name: featureName,
+  run: async context => {
+    try {
+      // Verify override path flag is provided
+      const outputDirParam = context.parameters.options ? context.parameters.options['output-dir'] : null;
+      if ( !outputDirParam ) {
+        throw new Error('Expected --output-dir flag to be set for model introspection command.');
```

Comment:
Do we want this to be an output dir, or an output path for the specific file?

Reply:
Confirmed with studio team. They agree with the output directory option with the hard-coded file name `model-introspection.json`. The name does not matter but it should keep unchanged.